governator-core/src/main/java/com/netflix/governator/ProvisionMetricsModule.java

```diff
@@ -53,7 +66,20 @@ protected void configure() {
         this.bindListener(Matchers.any(), listener);
         requestStaticInjection(MetricsProvisionListener.class);
```

Comment:
nope.  unfortunately there is no other way, that I know of.  This pattern is used successfully by many other components.  Note that no statics are actually created, this just forces early initialization.